content/en/agent/amazon_ecs/_index.md

```diff
@@ -66,7 +66,13 @@ Configure the task using either the [AWS CLI tools][12] or using the Amazon Web
 
 1. For Linux containers, download [datadog-agent-ecs.json][1] ([datadog-agent-ecs1.json][2] if you are using an original Amazon Linux 1 AMI). For Windows, download [datadog-agent-ecs-win.json][3].
 2. Edit `datadog-agent-ecs.json` and set `<YOUR_DATADOG_API_KEY>` with the [Datadog API key][4] for your account.
-3. Optionally - Add an Agent health check.
+3. Optionally - Add the following to you ECS task definition to deploy on an ["ECS-Anywhere" cluster][5].
```

Comment:
- just to double check, Is this optional or required on `"ECS-Anywhere" cluster` ? 
- could we also update the `Web UI` tab accordingly?

Reply:
Require for ECS-A

Reply:
Added the information on the `Web UI` tab.

Reply:
```suggestion
3. Optionally - Add the following to your ECS task definition to deploy on an [ECS Anywhere cluster][5].
```